packages/windows.treeinfo.json

```diff
@@ -1,5 +1,6 @@
 {
   "variants": {
+    "openssl",
```

Comment:
This needs to indicate that it uses the `windows` variant as with the other entries.

Reply:
ah... right...

Reply:
fixed